src/zui/ZUIDialog/index.tsx

```diff
@@ -30,6 +32,7 @@ const ZUIDialog: FunctionComponent<ZUIDialogProps> = ({
 
   return (
     <Dialog
+      disablePortal={disablePortal}
```

Comment:
Can you remind me the purpose of this, and is it still necessary if you move the provider?

Reply:
I've finally added a z-index property in the Dialog so no need for this